legateboost/__init__.py

```diff
@@ -1,28 +1,5 @@
-from .legateboost import LBClassifier, LBRegressor
-from .metrics import (
-    BaseMetric,
-    ExponentialMetric,
-    GammaDevianceMetric,
-    LogLossMetric,
-    MSEMetric,
-    NormalCRPSMetric,
-    NormalLLMetric,
-    GammaLLMetric,
-    QuantileMetric,
-)
-from .objectives import (
-    BaseObjective,
-    ExponentialObjective,
-    GammaDevianceObjective,
-    GammaObjective,
-    LogLossObjective,
-    NormalObjective,
-    QuantileObjective,
-    SquaredErrorObjective,
-)
-from .callbacks import (
-    TrainingCallback,
-    EarlyStopping,
-)
-from .utils import mod_col_by_idx, pick_col_by_idx, set_col_by_idx
+from .legateboost import *
+from .metrics import *
+from .objectives import *
+from .callbacks import *
```

Comment:
Did the linters complain about this? If not, I have a slight preference for putting back the specific imports instead of using star imports. For a few reasons:

1. it's helpful to be able to trace where an object is coming from just by using text search like `git grep` or similar
2. helps IDEs that might not be able to follow star imports (I have no specific examples of this, to be fair)
3. might make the source of name conflicts (accidentally importing the same name from 2 sub-modules) easier to identify

All of this is personal preference... I can't think of any **correctness** reason why star imports would be a problem given the other changes in this PR, and I do appreciate the appeal of only having to manage this "what is public" information in one place.

So do whatever you want with this suggestion, just wanted to make my case.

Reply:
I don't like writing the same thing in multiple places so I'd rather keep it the way it is for now.

Reply:
sure, works for me!